readme.md

```diff
@@ -8,28 +8,46 @@
 
 ## Overview
 
+Chief-of-state (COS) is a dockerized persistence framework for building modern, clustered, reactive applications.
```

Comment:
I think the most important aspects of COS are that it's an event sourced persistence framework that scales, so maybe:

Chief-of-state (COS) is a ~dockerized~ clustered persistence framework for building ~modern, clustered, reactive~  event sourced applications.

Reply:
Yeah that's similar to what we had before. I'm partial to event sourcing, but do you think that's too prescriptive?